app/src/main/res/layout/state_fragment.xml

```diff
@@ -15,20 +15,31 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
     <LinearLayout
+      android:id="@+id/linearLayout"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="24dp"
       android:layout_marginEnd="24dp"
       android:visibility="@{viewModel.isAudioFragmentVisible? View.VISIBLE: View.GONE}"
       app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent">
+      app:layout_constraintStart_toStartOf="parent" app:layout_constraintTop_toTopOf="parent">
```

Comment:
Why combine these into one line? Prefer attributes to be on their own line.

Reply:
Fixed attributes in their own line.